spec/suites/SpecHelper.js

```diff
@@ -79,5 +79,33 @@ var touchEventType = L.Browser.touchNative ? 'touch' : 'pointer'; // eslint-disa
 // Note: this override is needed to workaround prosthetic-hand fail,
 //       see https://github.com/Leaflet/prosthetic-hand/issues/14
 
+function createContainer() {

```

Comment:
We could also add optional `width` and `height` parameters here to cut down on boilerplate.